server/src/main/java/org/opensearch/gateway/remote/RemoteClusterStateService.java

```diff
@@ -500,6 +578,15 @@ private static String indexMetadataFileName(IndexMetadata indexMetadata) {
         );
     }
 
+    private static String globalMetadataFileName(Metadata metadata) {
```

Comment:
Thanks for highlighting, added version in file name. We will look at this version for BWC in future modification.

Reply:
which version are you picking here? There is already cluster state version in the manifest file

Reply:
@shwetathareja , it is not cluster state version, it is new version we have introduced, we can think of it as codec version. 

For now version is hard coded to `1`, in future if we want to change implementation of global metadata[let's say move some metadata to separate files instead of global metadata] we will bump up the version. We can leverage this version to maintain backward compatibility in future changes with older files.